client/aliases.go

```diff
@@ -81,6 +81,8 @@ func (client *Client) Unalias(aliasOrSnap string) (changeID string, err error) {
 type AliasStatus struct {
 	App    string `json:"app,omitempty"`
 	Status string `json:"status,omitempty"`
```

Comment:
Do we still need Status, or should we have a structure that more closely resembles that of the internal representation, with aliases and a disabled flag in the Snap proper?  Okay for a follow up if that sounds better.